### PR TITLE
Add basic structure

### PR DESCRIPTION
**Pull request by QuanXuuu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,60 @@
...
+        </p>
+
+        <button class="card__button__button-show">show answer</button>
+        <p>Answer: Lorem ipsum dolor sit amet.</p>
+
+        <ul role="list" class="tags">
+          <li class="tags__tag1">tag one</li>
+          <li class="tags__tag2">tag two</li>
+          <li class="tags__tag3">tag three</li>
+          <li class="tags__tag4">tag four</li>
```
Comment (ahohnsen) [inline index.html:31]: If all your tags will have the same styling, you would need only one class for all of them :)

File: index.html [hunk 2]
```diff
@@ -4,9 +4,60 @@
...
+      </section>
+
+      <section class="card">
+        <h2 class="card__headline">Question</h2>
+        <p class="card__question">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Maiores sunt
+          possimus debitis neque laboriosam voluptas error.
+        </p>
+
+        <button class="card__button__button-show">show answer</button>
```
Comment (ahohnsen) [inline index.html:42]: Following BEM I would suggest to change the class name: `card__button--show`

File: routes/bookmarks.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+</header>
+
+  <section class="card">
+    <h2 class="card__headline">Question</h2>
+    <p class="card__question">
+      Lorem ipsum dolor sit, amet consectetur adipisicing elit. Maiores sunt
+      possimus debitis neque laboriosam voluptas error.
+    </p>
+
+    <button class="card__button__button-show">show answer</button>
```
Comment (ahohnsen) [inline routes/bookmarks.html:21]: Same as in your index.html :)

File: routes/profile.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        </p>
+
+        <ul role="list" class="skills">
+          <li class="skills__skill1">skill_one</li>
+          <li class="skills__skill2">skill_two</li>
+          <li class="skills__skill3">skill_three</li>
+          <li class="skills__skill4">skill_four</li>
+        </ul>
+
+        <button class="button__button-logout">logout</button>
```
Comment (ahohnsen) [inline routes/profile.html:33]: Here I would suggest as well to change the class name (following BEM): `button--logout`

**Review by ahohnsen — APPROVED** [review]
Well done :) I have a few minor suggestions for you, see my comments.

File: index.html (hunk 1 above)
Comment (QuanXuuu) [reply index.html]: Thank you! changed already :)

File: css/card.css [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+.card__bookmark--regular {
+  background-color: white;
+}
+
+.card__button--show {
+  background-color: rgb(219, 218, 218);
+  border-radius: 5px;
+  padding: 8px;
```
Comment (beetlejays) [inline css/card.css:42]: hier kannst du vielleicht rem nehmen anstatt pixel :)

File: js/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+const button = document.querySelector("[data-js='button']");
+
+function showAnswer() {
+  const answer = document.querySelector("[data-js='answer']");
+  answer.classList.toggle("card__answer");
+
+  if (this.innerText === "Show Answer") {
```
Comment (beetlejays) [inline js/index.js:7]: Cool wenn du .this nutzt, war für mich bisher nie richtig klar :)

**Pull request merged**